elements/pfe-button/src/pfe-button.js

```diff
@@ -0,0 +1,192 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+// @IE11
+// watching for addition and removal of nodes so
+// we can make sure we have the correct light DOM
+// and so we can set the _externalBtn property
+const parentObserverConfig = {
+  childList: true
+};
+
+// watching for changes on the _externalBtn so we can
+// update text in our shadow DOM when the _externalBtn
+// changes
+const externalBtnObserverConfig = {
+  characterData: true,
+  attributes: true,
+  subtree: true,
+  childList: true
+};
+
+// list of attributes that we DO NOT want to pass from
+// the _externalBtn to our shadow DOM button. For example,
+// the style attribute could ruin our encapsulated styles
+// in the shadow DOM
+const blackListedAttributes = ["style"];
```

Comment:
Suggest: `denylistAttributes`

Reply:
Follow some of the naming here: https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language

`allowlist` and `denylist`